scripts/set_trustlines.ts

```diff
@@ -0,0 +1,78 @@
+import axios from 'axios';
+import { config } from '../utils/env_config.js';
+import * as StellarSdk from 'stellar-sdk';
+import { TokensBook } from '../utils/tokens_book.js';
+
+export interface TokenType {
+  address: string;
+  name: string;
+  symbol: string;
+  decimals?: number;
+  logoURI?: string;
+}
+
+export interface tokensResponse {
+  network: string;
+  tokens: TokenType[];
+}
+
+export async function setTrustline(tokenSymbol: string, tokenIssuer: string) {
+  let source = await loadedConfig.horizonRpc.loadAccount(process.env.TRUSTLINE_WALLET_PUBLIC_KEY!);
```

Comment:
We can get public key from the private key.
`Keypair.fromSecret(userSecretKey).publicKey()`

https://stellar.github.io/js-stellar-sdk/Keypair.html